.github/workflows/pr-ci.yml

```diff
@@ -1,32 +1,44 @@
 name: CI build and push (PR)
 
 concurrency:
-  group: pr-${{ github.event.number }}
+  group: ci-${{ github.run_id }}
   cancel-in-progress: true
 
 on:
-  pull_request_target:
-    types:
-      - opened
-      - reopened
-      - synchronize
-      - labeled
-      - unlabeled
-    branches:
-      - main
-      - v[0-9]+
-      - v[0-9]+.[0-9]+
-      - cryostat-v[0-9]+.[0-9]+
+  issue_comment:
+    types: [created]
 
 jobs:
   code-analysis:
-    uses: ./.github/workflows/ci-code-analysis.yml
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'cryostatio' && contains(github.event.pull_request.labels.*.name, 'safe-to-test') && github.event.issue.pull_request && startsWith(github.event.comment.body, '/build_test')
+    steps:
+    - name: React to comment
+    uses: actions/github-script@v4
     with:
-      checkout-repo: ${{ github.event.pull_request.head.repo.full_name }}
-      checkout-ref: ${{ github.event.pull_request.head.ref }}
-    secrets: inherit
-    if: github.repository_owner == 'cryostatio' && contains(github.event.pull_request.labels.*.name, 'safe-to-test')
-
+      script: |
+          const {owner, repo} = context.issue
+          github.reactions.createForIssueComment({
+            owner,
+            repo,
+            comment_id: context.payload.comment.id,
+            content: "+1",
+          });  
+    - uses: tspascoal/get-user-teams-membership@v2
+      id: checkMember
+      with:
+        username: ${{ github.actor }}
+        team: 'cryostat/reviewers'
+        GITHUB_TOKEN: ${{ secrets.PAT }}
+    - name: Fail if not a Cryostat reviewer
```

Comment:
Can we grab this checking step, along with the `safe-to-test` and `needs-triage` label check into a separate job? Then, I think other build and push jobs can just depends on it.